src/main/java/org/openRealmOfStars/player/tech/TechFactory.java

```diff
@@ -545,6 +545,9 @@ public static Tech createCombatTech(final String name, final int level) {
         } else if (techName.startsWith("Tractor beam")) {
           tech.setIcon(Icons.getIconByName(Icons.ICON_TRACTOR_BEAM));
           tech.setRareTech(true);
+        } else if (techName.startsWith("Callisto multicannon Mk1")) {
```

Comment:
Also after adding multiple tech of diffrent levels, remove the Mk1 from string so it matches all of them if they start with similar name.